src/emails/user-invitation.hbs

```diff
@@ -0,0 +1,19 @@
+---
+layout: default.hbs
+subject: You’ve been invited to GoldSilver.com
+preheader: Please set a password to sign into your new account.
+---
+<table width="100%" cellpadding="0" cellspacing="0">
+    <tr>
+        <td class="content-block">
+            This is a stub
+        </td>
+    </tr>
+    <tr>
+        <td class="content-block">
+            <h1>You’ve been invited to GoldSilver.com.</h1>
+            <p>To sign in and use your new account, please create a password using the link below:</p>
+            <p><a href="">\{{ set password link }}</a></p>
```

Comment:
variable should be in href attribute, text can be hardcoded to 'Reset my password'